pkg/selector/selector_test.go

```diff
@@ -657,10 +687,12 @@ func TestFilter_PricePerHour_NoResults(t *testing.T) {
 
 func TestFilter_PricePerHour_OD(t *testing.T) {
 	ec2Mock := setupMock(t, describeInstanceTypesPages, "t3_micro.json")
+	now := time.Now()
 	itf := selector.Selector{
 		EC2: ec2Mock,
 		EC2Pricing: &ec2PricingMock{
 			GetOndemandInstanceTypeCostResp: 0.0104,
+			lastOnDemandCacheUTC:            &now,
```

Comment:
By ensuring the `*CacheUTC` field(s) are populated with a non-`nil` value, the tests exercise relevant code paths in `selector.rawFilter`.